curriculum/challenges/english/25-front-end-development/workshop-merge-sort/6569b20f829b7e69d43c232a.md

```diff
@@ -0,0 +1,60 @@
+---
+id: 6569b20f829b7e69d43c232a
+title: Step 21
+challengeType: 20
+dashedName: step-21
+---
+
+# --description--
+
+The last thing to do for the `while` loop is to move to the next index in the sorted array.
+
+Using the augmented assignment addition operator, add `1` to the value of `sorted_index`.
```

Comment:
Probably is not necessary to require the use of `+=`. Also, the test accepts other valid code.